spec/features/object_resource_spec.rb

```diff
@@ -244,6 +244,34 @@
     end
   end
 
+  context 'IIIF OpenSeaDragon viewer' do
+    let(:user) { create(:admin) }
+    let(:object_with_file) { FactoryGirl.create(:object_with_image_file, title: ["Object - IIIF Viewer"], user: user) }
+
+    before do
```

Comment:
why is this needed? or rather, what would be the downside of just setting `iiif_enabled` to `true` in the [test environment](https://github.com/ucsdlib/horton/blob/a8f133a7f31417ece7f71c73c9d5b6433de04870/config/environments/test.rb#L62)? it seems you could do away with this entirely.

Related, if we set it to true, then it seems we'd have that value set to `true` for all environments, which makes me wonder what the need for the configuration setting is, and the related checks for whether it's true or not, such as [this](https://github.com/ucsdlib/horton/blob/a8f133a7f31417ece7f71c73c9d5b6433de04870/app/views/hyrax/base/show.html.erb#L11)

Reply:
@mcritchlow I think the original thought is enable us to hide the viewer for any purpose. But since now we've added the mouse over trigger to have it shown up, so I think we can remove it.
I've updated the PR to remove the configuration: https://github.com/ucsdlib/horton/pull/138/commits/a975b474b0711dd81d4c852a74939e7a871c585f.

Reply:
Ah, ok. Yeah that sounds good. Thanks! @VivianChu do you have any other questions or concerns, or should we get this merged?

Reply:
@mcritchlow - I don't have any other question.  We should get this merged.